### PR TITLE
eslint can suck my dick

### PR DESCRIPTION
**Pull request by d3rpp**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [*]
 # gotta be friendly to windows
-end_of_line = crlf
+end_of_line = unset
```
Comment (dylhack) [inline .editorconfig:3]: should be lf frick windows

File: .eslintrc.json [hunk 2]
```diff
@@ -3,21 +3,21 @@
...
     "rules": {
+        "newline-before-return": "off",
         "import/no-unresolved": "off",
         "import/extensions": "off",
         "class-methods-use-this": "off",
-        "no-console": "off"
+        "no-console": "off",
+        "linebreak-style": "error",
+        "quotes": "off",
+        "no-undef": "off"
```
Comment (dylhack) [inline .eslintrc.json:21]: use null

File: .eslintrc.json [hunk 3]
```diff
@@ -3,21 +3,21 @@
...
+    "plugins": ["@typescript-eslint"],
     "rules": {
+        "newline-before-return": "off",
         "import/no-unresolved": "off",
         "import/extensions": "off",
         "class-methods-use-this": "off",
-        "no-console": "off"
+        "no-console": "off",
+        "linebreak-style": "error",
+        "quotes": "off",
```
Comment (dylhack) [inline .eslintrc.json:20]: why disable quotes enforcing?

File: prisma/schema.prisma [hunk 4]
```diff
@@ -23,3 +23,7 @@ model LastChecked {
     guildId String      @id
     guild   GuildConfig @relation(fields: [guildId], references: [guildId])
 }
+
+model CheckPost {
```
Comment (dylhack) [inline prisma/schema.prisma:27]: remove this from the PR

File: src/database.ts [hunk 5]
```diff
@@ -1,65 +1,65 @@
...
-	return config;
-}
+import { GuildConfig, PrismaClient } from "@prisma/client";
+import { createClient, RedisClientType } from "redis";
+import { getRedisUrl } from "./config";
+
+let prismaInstance: PrismaClient | null = null;
+let redisInstance: RedisClientType | null = null;
+
+export function getPrismaClient(): PrismaClient {
```
Comment (dylhack) [inline src/database.ts:8]: doesn't need to be exported

File: src/database.ts [hunk 6]
```diff
@@ -1,65 +1,65 @@
...
+  if (prismaInstance !== null) {
+    return prismaInstance;
+  }
+  prismaInstance = new PrismaClient();
+  return prismaInstance;
+}
+
+export function getRedisClient(): RedisClientType {
```
Comment (dylhack) [inline src/database.ts:16]: doesn't need to be exported

**Review by dylhack — CHANGES_REQUESTED** [review]
did you mean to leave in wordpress code with eslint related PR stuff?

File: src/database.ts (hunk 5 above)
Comment (d3rpp) [reply src/database.ts:8]: how tf else am i supposed to connect to the db?

File: .eslintrc.json (hunk 3 above)
Comment (d3rpp) [reply .eslintrc.json:20]: eslint wants single quotes for imports
editor wants double quoted for imports

I get errors either way

File: src/database.ts (hunk 6 above)
Comment (d3rpp) [reply src/database.ts:16]: how tf else am i supposed to connect to redis?

File: .eslintrc.json (hunk 3 above)
Comment (dylhack) [reply .eslintrc.json:20]: sadge okay

**Comment by d3rpp** [comment]
This is now a walking talking merge conflict

I'll reclone this and make my changes because this is fucked

**Pull request closed without merging**